core/src/main/java/org/apache/carbondata/core/metadata/schema/table/CarbonTable.java

```diff
@@ -152,6 +154,15 @@ public static CarbonTable buildFromTableInfo(TableInfo tableInfo) {
     }
     table.hasDataMapSchema =
```

Comment:
Please don't change the carbonTable for aggdatamap. Add a utility function to decide it has agg datamap or not